docs(breadcrumb, checkbox, clearbutton, coachindicator): Site Docs to Storybook (#2793)
adobe/spectrum-css: components/coachindicator/stories/coachindicator.mdx

import { Canvas, ArgTypes, Meta, Description, Title } from '@storybook/blocks';

import * as CoachIndicator from './coachindicator.stories';

<Meta of={CoachIndicator} title="Docs" />

<Title of={CoachIndicator} />
<Description of={CoachIndicator} />

> ### Migration Guide
> 
> Coach Indicator is now a separate component from [Coach Mark.](/docs/components-coach-mark--docs)

Reviewer: ```suggestion
Coach Indicator is primarily used along with the [Coach Mark](/docs/components-coach-mark--docs) component.
```

Reviewer 2: Hehehe suggestion inception! Reminder to use sentence case even for component names

```suggestion
### Migration Guide

Coach indicator is now a separate component from [Coach mark.](/docs/components-coach-mark--docs)
```

Author: great catches from both! https://github.com/adobe/spectrum-css/pull/2793/commits/6b0faaeb26da1a3fb56c138f05f1dcbf436a59ff

import { Canvas, ArgTypes, Meta, Description, Title } from '@storybook/blocks';

import * as CoachIndicator from './coachindicator.stories';

<Meta of={CoachIndicator} title="Docs" />

<Title of={CoachIndicator} />
<Description of={CoachIndicator} />

### Migration Guide

> Coach indicator is primarily used along with the [Coach mark](/docs/components-coach-mark--docs) component.

Reviewer 3: This seems like less of a migration guide and more of how to use the component. I think you could update the description for the component to this and successfully remove the mdx file to let the template pick-up.
